nengo/utils/numpy.py

```diff
@@ -57,7 +57,7 @@ def array(x, dims=None, min_dims=0, readonly=False, **kwargs):
     return y
 
 
-def array_hash(a, n=100):
+def array_hash(a, n=10000):
```

Comment:
Note to reviewers: I changed this limit because hashing an array of this size is still very fast on my machine (~0.1 ms). The previous limit was just too small, resulting in more possibility for hash collisions.